### PR TITLE
Newbuildtest

### PR DESCRIPTION
**Pull request by SusmithaGundu**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -52,11 +52,17 @@ subprojects {
...
+      }
       compileOnly "org.projectlombok:lombok"
-      annotationProcessor platform("io.spinnaker.kork:kork-bom:$korkVersion")
+      annotationProcessor platform("io.spinnaker.kork:kork-bom:$korkVersion"){
+        exclude group: 'org.apache.groovy', module: 'groovy-json'
+      }
       annotationProcessor "org.projectlombok:lombok"
-      testAnnotationProcessor platform("io.spinnaker.kork:kork-bom:$korkVersion")
+      testAnnotationProcessor platform("io.spinnaker.kork:kork-bom:$korkVersion"){
+        exclude group: 'org.apache.groovy', module: 'groovy-json'
```
Comment (kirangodishala) [inline build.gradle:64]: why are we excluding groovy-json?

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy [hunk 2]
```diff
@@ -202,6 +202,8 @@ class BuildController {
       .dropRight(2)
       .join('/')
 
+    def var = request.getRequestURI().split('/')
+    buildNumber = var[var.length-1].toInteger()
```
Comment (kirangodishala) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy:206]: can you add a comment to show where exactly the buildNumber come from in the request mapping?

File: igor-web/src/test/groovy/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildTest.java [hunk 3]
```diff
@@ -377,3 +384,4 @@ private Operation operationResponse() {
     return operation;
   }
 }
+ */
```
Comment (kirangodishala) [inline igor-web/src/test/groovy/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildTest.java:387]: Why is this class completely commented? Shouldn't we make the tests pass?

File: build.gradle (hunk 1 above)
Comment (SusmithaGundu) [reply build.gradle]: reverted the exclusion of groovy-json package

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy (hunk 2 above)
Comment (SusmithaGundu) [reply igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy:206]: Added the comment

**Review by kirangodishala — DISMISSED** [review]
LGTM

**Comment by kirangodishala** [comment]
@SusmithaGundu - looks like there are some conflicts...

**Comment by SusmithaGundu** [comment]
Resolved build conflicts

**Review by sanopsmx — APPROVED** [review]

**Pull request merged**